peas/weather.py

```diff
@@ -604,20 +587,19 @@ def get_wind_speed(self, n=3):
         return self.wind_speed
 
     def send_message(self, msg, channel='weather'):
-        if self.messaging is None:
-            self.messaging = PanMessaging.create_publisher(6510)
-
-        self.messaging.send_message(channel, msg)
+        super.send_message()
 
     def capture(self, use_mongo=False, send_message=False, **kwargs):
         """ Query the CloudWatcher """
 
-        self.logger.debug("Updating weather")
+        self.logger.debug("Updating weather data")
 
         data = {}
-        data['weather_sensor_name'] = self.name
+
+        data['weather_data_from'] = self.cfg.get('product_1')
```

Comment:
You'll want to pull from the config once and then use that as a variable everywhere else. So this should be done in `init` probably.